spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/LoadBalancerProperties.java

```diff
@@ -37,6 +38,16 @@
 @ConfigurationProperties("spring.cloud.loadbalancer")
 public class LoadBalancerProperties {
 
+	private Map<String, LoadBalancerProperties> instances = new HashMap<>();
```

Comment:
Let's rename this field to sth more meaningful (`instance` is a term we use in a very specific way), maybe `clientProperties`, `serviceProperties`, etc.?